src/ooni/http_invalid_request_line.cpp

```diff
@@ -0,0 +1,126 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <measurement_kit/http.hpp>
+#include <measurement_kit/ooni.hpp>
+
+#include "src/common/utils.hpp"
+
+namespace mk {
+namespace ooni {
+
+static const int timeout = 5;
```

Comment:
Used to be a class attribute, could perhaps moved in settings if we want to set it